Mutagen.Bethesda.Analyzers.Skyrim/Record/Armor/VendorKeywordAnalyzer.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Mutagen.Bethesda.Analyzers.SDK.Analyzers;
+using Mutagen.Bethesda.Analyzers.SDK.Results;
+using Mutagen.Bethesda.Analyzers.SDK.Topics;
+using Mutagen.Bethesda.Plugins;
+using Mutagen.Bethesda.Skyrim;
+using Noggog;
+namespace Mutagen.Bethesda.Analyzers.Skyrim.Record.Armor;
+
+public class VendorKeywordAnalyzer : IIsolatedRecordAnalyzer<IArmorGetter>
+{
+    public static readonly TopicDefinition<string, string> ArmorMissingVendorKeyword = MutagenTopicBuilder.DevelopmentTopic(
+            "Armor is missing Vendor Keyword",
+            Severity.Suggestion)
+        .WithFormatting<string, string>("Missing vendor keyword {0} but has {1}");
```

Comment:
Asking more questions, just to figure out how things fit into categories better.

Does this cause an issue in game?  Or is this just typically the pattern that things should have to make sense?  What happens when an armor is missing the vendor keyword?

Reply:
It means that you can't sell the armor in game. The way vendors function is fully keyword drive. There is a vendor X who buys all items that have keyword Y. So only in very few cases this shouldn't be the case.